Library/Formula/c.rb

```diff
@@ -0,0 +1,13 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.09.tar.gz"
+  sha256 "e22bc962ba4ed9a1d84929670626abf7372271efc9d478899fef9981449382ab"
+
+  def install
+    bin.install "c"
+  end
+
+  test do
+    system "echo", "int main(void){return 0;}", "|c"
```

Comment:
I'll happily admit that I do not know anything about ruby (and seeing this, I don't really want to deal with it) – my guess was that I would need to write `system "echo", … "\"|c\""` to get a string instead of a pipe. I have updated the test accordingly. Even though, I fail to see any reason to use a source code file instead of a simple `echo` and from other formula's tests I gather that the path should be read from a variable as `#{bin}` instead of being literal `bin/`.